packages/web-components/src/components/masthead/masthead-container.ts

```diff
@@ -152,17 +168,36 @@ const defaultUnauthenticateProfileItems: MastheadProfileItem[] = [
   },
 ];
 
+/**
+ * @param props A key/value pair.
+ * @returns The modified version of the given `props` with all properties with `undefined` value removed.
+ */
+function cleanProps(props: { [key: string]: unknown }) {
```

Comment:
Should this eventually be a utility?

Reply:
It's a possibility. Wanted to see if this approach (clean afterwards) is the best, or there is a better way to prevent redundant props defined here.